extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/internal/handler/ZonePlayerHandler.java

```diff
@@ -1541,33 +1554,37 @@ private SonosZoneGroup getCurrentZoneGroup() {
 
     /**
      * Sets the volume level for a notification sound
-     * (initializes {@link #notificationSoundVolume})
      *
-     * @param command
+     * @param notificationSoundVolume
      */
-    public void setNotificationSoundVolume(Command command) {
-        if (command != null) {
-            notificationSoundVolume = command.toString();
+    public void setNotificationSoundVolume(PercentType notificationSoundVolume) {
+        if (notificationSoundVolume != null) {
+            logger.debug("Sonos device {}: Update configuration for 'notificationVolume' to {}", getThing().getUID(),
+                    notificationSoundVolume);
+            Configuration config = editConfiguration();
+            config.put(ZonePlayerConfiguration.NOTIFICATION_VOLUME, notificationSoundVolume.toBigDecimal());
+            updateConfiguration(config);
```

Comment:
This is imho pretty nasty and hints at the fact that this is rather a hack than a recommended way to model things. I would vote for removing the channel completely, see my https://github.com/eclipse/smarthome/issues/4832#issuecomment-355505603.